pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -24,16 +24,16 @@ import (
 	"sort"
 	"strings"
 
+	v1 "k8s.io/api/core/v1"
```

Comment:
same here, the v1 import alias is redundant :)

Reply:
addressed